src/main/features/core/lyrics/source_musiXmatch.js

```diff
@@ -0,0 +1,42 @@
+import { AllHtmlEntities as Entities } from 'html-entities';
+import fetch from 'node-fetch';
+import xss from 'xss';
+import JSON from 'JSON';
+
+const decoder = new Entities();
+
+const attemptMusiXmatch = (path) => {
+  return new Promise((resolve, reject) => {
+    fetch(`https://www.musixmatch.com/search/${path}`) // the lyrics urls aren't always very straightforward...
+      .then((data) => data.text())
+      .then((searchHtml) => {
+        const searchReturn = JSON.parse(/__mxmProps = ({.+})<\/script>/.exec(searchHtml)[1]);
+        fetch(searchReturn.allResults.bestMatch.shareURI)
```

Comment:
@MCManuelLP I'm saying that letting a promise fail due to a syntax / execution error is akin to (same as) wrapping your entire code in a try catch.  Not that you should do that, in fact, the complete opposite.  I'm saying you should never wrap your code in a try catch and let it fail.  Rather check for failures / potential errors specifically at teach step.

However in this case I believe it is OK as any failure inside this method should simply reject the promise and by nature of JS errors that is what happens :+1: